### PR TITLE
PBI-6387-RichContent

### PR DESCRIPTION
**Pull request by PoornimaApptentive**
[PBI-6387](https://alchemer.atlassian.net/browse/PBI-6387)

multiuser support for Android
-- login, logout, update token and AuthroizationFailureListener
multiuser support for iOS
-- login & logout
6.6 for iOS
6.7 for Android

[PBI-6387]: https://alchemer.atlassian.net/browse/PBI-6387?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: example/ios/Flutter/AppFrameworkInfo.plist [hunk 1]
```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>11.0</string>
+  <string>12.0</string>
```
Comment (frankus) [inline example/ios/Flutter/AppFrameworkInfo.plist:24]: This should be 13.0

File: example/ios/Podfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-platform :ios, '11.0'
+platform :ios, '12.0'
```
Comment (frankus) [inline example/ios/Podfile:2]: Also should be 13.0

File: example/ios/Flutter/AppFrameworkInfo.plist (hunk 1 above)
Comment (frankus) [reply example/ios/Flutter/AppFrameworkInfo.plist:24]: Need to look at what generates these (in any case not your fault :)

File: example/lib/main.dart [hunk 3]
```diff
@@ -46,8 +46,8 @@ class _MyAppState extends State<MyApp> {
     final String apptentiveKey;
     final String apptentiveSignature;
     if (Platform.isAndroid) {
-      apptentiveKey = "<YOUR_ANDROID_KEY>";
-      apptentiveSignature = "<YOUR_ANDROID_SIGNATURE>";
+      apptentiveKey = "ANDROID-FOXFAX-d2c49794fc25";
```
Comment (frankus) [inline example/lib/main.dart]: Will want to remove this from the branch. Not sure if it's worth the effort to completely purge it from GitHub (involves a support request).

**Review by frankus — COMMENTED** [review]
Looks good, wondering if it's worth manually editing the files for the example app for an iOS 13+ deployment target. When I looked at this recently Flutter was still setting it to 12.0 when rebuilding.

**Review by frankus — APPROVED** [review]

File: example/lib/main.dart (hunk 3 above)
Comment (PoornimaApptentive) [reply example/lib/main.dart]: Updated with the latest commit.

File: example/ios/Flutter/AppFrameworkInfo.plist (hunk 1 above)
Comment (PoornimaApptentive) [reply example/ios/Flutter/AppFrameworkInfo.plist]: For example app it was automatically set to 12. However, FoxFacts didn't like to be at 13. It was throwing some build errors.

File: pubspec.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 name: apptentive_flutter
 description: Apptentive SDK for Flutter
-version: 6.2.2
+version: 6.7.0
```
Comment (PoornimaApptentive) [inline pubspec.yaml:3]: I intentionally didn't bump the version as I wasn't aware if it is part of the release process as we used to do with the SDK builder. But we can use this PR for release as well.

Comment (frankus) [reply pubspec.yaml:3]: I'll have to refresh my memory how everything works, but I'm hoping we can use the GitHub actions to release this time. In any case, it probably would be good to leave this at 6.2.2 so we can see the diff in the release.

**Pull request merged**